src/app/core/cases/case-view-event.model.ts

```diff
@@ -10,4 +10,9 @@ export class CaseViewEvent {
   user_id: number;
   user_last_name: string;
   user_first_name: string;
+  significant_item: {
```

Comment:
if we make significant item optional we won't need this and the below additions to the CaseView object...